airbyte-integrations/connectors/source-mongodb-v2/integration_tests/expected_spec.json

```
@@ -108,13 +108,19 @@
         "airbyte_secret": true,
         "order": 3
       },
+      "level": {
```

```suggestion
      "level": {
```
Can you give a more describable name? Maybe `max_depth_level_read`